stl/inc/forward_list

```diff
@@ -242,8 +243,9 @@ struct _Flist_node { // forward_list node
     using _Nodeptr = _Rebind_pointer_t<_Voidptr, _Flist_node>;
 
     _Nodeptr _Next; // successor node
-    _Value_type _Myval; // the stored value
+    _Value_type _Myval = _Returns_exactly<_Value_type>(); // the stored value
 
+    _Flist_node()                   = default;

```

Comment:
This is well past the threshold of cleverness that we need a comment to explain what's going on. It's doubly important given that we've got no testing in place to catch fire if someone sees this silliness and decides to revert it. Something like:
```suggestion
    _Value_type _Myval = _Returns_exactly<_Value_type>(); // Fake a viable constructor to workaround GH-2740

    _Flist_node()                   = default;
```
in all three places. (Yes, I recommend removing the existing low-value field comments.)